contracts/upgradeability/AdminUpgradeabilityProxy.sol

```diff
@@ -21,7 +21,8 @@ contract AdminUpgradeabilityProxy is UpgradeabilityProxy {
   event AdminChanged(address previousAdmin, address newAdmin);
 
   // Storage slot of the admin of the contract
-  bytes32 private constant ADMIN_SLOT = keccak256("org.zeppelinos.proxy.admin");
+  bytes32 private constant ADMIN_SLOT =
+    0x10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b;
```

Comment:
Nit: it's weird to have multiple-line assignments.
I'd a comment here saying this value is `keccak256("org.zeppelinos.proxy.admin")` (I know we are asserting that in the `constructor`)

Reply:
@facuspagnuolo but why this changes were made?

Reply:
Just got it.